theme/lumo/vaadin-radio-button-css.ts

```diff
@@ -0,0 +1,116 @@
+import { css } from 'lit-element';
+import '@vaadin/vaadin-lumo-styles/color.js';
+
+export const radioButtonStyles = css`
+  :host {
+    -webkit-tap-highlight-color: transparent;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    cursor: default;
+    outline: none;
+  }
+
+  :host(:not([label-empty])) [part='label'] {
+    margin: 0.1875em 0.875em 0.1875em 0.375em;
+  }
+
+  [part='radio'] {
+    width: calc(1em + 2px);
+    height: calc(1em + 2px);
+    margin: 0.1875em;
+    position: relative;
+    border-radius: 50%;
+    background-color: var(--lumo-contrast-20pct);
+    transition: transform 0.2s cubic-bezier(0.12, 0.32, 0.54, 2), background-color 0.15s;
+    pointer-events: none;
+    will-change: transform;
+    line-height: 1.2;
+  }
+
+  /* Used for activation "halo" */
+  [part='radio']::before {
+    /* Needed to align the radio-button nicely on the baseline */
+    content: '\\2003';
+    color: transparent;
+    display: inline-block;
+    width: 100%;
+    height: 100%;
+    border-radius: inherit;
+    background-color: inherit;
+    transform: scale(1.4);
+    opacity: 0;
+    transition: transform 0.1s, opacity 0.8s;
+    will-change: transform, opacity;
+  }
+
+  /* Used for the dot */
+  [part='radio']::after {
+    content: '';
+    width: 0;
+    height: 0;
+    border: 3px solid var(--lumo-primary-contrast-color);
+    border-radius: 50%;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%) scale(0);
+    transition: 0.25s transform;
+    will-change: transform;
+    background-clip: content-box;
+  }
+
+  :host([checked]) [part='radio'] {
+    background-color: var(--lumo-primary-color);
+  }
+
+  :host([checked]) [part='radio']::after {
+    transform: translate(-50%, -50%) scale(1);
+  }
+
+  :host(:not([checked]):not([disabled]):hover) [part='radio'] {
```

Comment:
Missing `:not([indeterminate])`. Was it changed on purpose?

Reply:
We don't have `indeterminate` in radio button, it was a copy-paste from checkbox.